github-profiles/Leal2-6465.md

```diff
@@ -0,0 +1,6 @@
+# My github profile  https://github.com/Lealdos
```

Comment:
This file shouldn't exist, because it has been move into the folder instead.

Reply:
Hello Guillermo, I fixed the error and deleted the leftover file


![image](https://user-images.githubusercontent.com/105816982/224592975-d93564cc-dc16-48c3-9271-eb0e3eeb6297.png)